packages/cli/src/commands/deployment/deploy.ts

```diff
@@ -75,9 +41,9 @@ export default class Deploy extends Command {
     projectName = await valueOrPrompt(projectName, 'Enter project name', 'Project name is required');
     ipfsCID = await valueOrPrompt(ipfsCID, 'Enter IPFS CID', 'IPFS CID is required');
 
-    const validator = await ipfsCID_validate(ipfsCID, authToken, ROOT_API_URL_PROD);
-    queryVersion = addV(queryVersion);
-    indexerVersion = addV(indexerVersion);
+    const validator = await ipfsCID_validate(String(ipfsCID), authToken, ROOT_API_URL_PROD);
+    queryVersion = addV(String(queryVersion));
+    indexerVersion = addV(String(indexerVersion));
```

Comment:
Why do these now need to be cast to string? There are multiple cases where this is now happening

Reply:
The typing from oclif is very different in structure. Made for forced type conversion only

Reply:
reverted.